recipe/meta.yaml

```
@@ -1,52 +1,58 @@
 {% set name = "pycocotools" %}
-{% set version = "2.0.6" %}
+{% set version = "2.0.8" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7fe089b05cc18e806dcf3bd764708d86dab922a100f3734eb77fb77a70a1d18c
+  sha256: 8f2bcedb786ba26c367a3680f9c4eb5b2ad9dccb2b34eaeb205e0a021e1dfb8d
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-build-isolation -vvv
+  skip: true  # [py<39]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
   host:
     - python
     - pip
     - setuptools
+    - wheel
     - cython
     - matplotlib-base
     - numpy
   run:
     - python
-    - cython
-    - matplotlib-base
     - {{ pin_compatible('numpy') }}
 
 test:
   imports:
     - pycocotools
     - pycocotools.mask
+  commands:
+    - pip check
+  requires:
+    - python
+    - pip
+    - numpy
+    - matplotlib-base
```

```suggestion
    - pip
```
matplotlib-base, numpy, and python are in run so there is no need to duplicate them in test/requires